src/test/erasure-code/CMakeLists.txt

```diff
@@ -83,9 +83,12 @@ target_link_libraries(unittest_erasure_code_plugin_jerasure
   global
   osd
   ec_jerasure
-  common)
+  common
+  ${UNITTEST_LIBS}
+  )
 add_dependencies(unittest_erasure_code_plugin_jerasure
   ec_jerasure)
+set_target_properties(unittest_erasure_code_plugin_jerasure PROPERTIES COMPILE_FLAGS ${UNITTEST_CXX_FLAGS})
```

Comment:
@tchaikov 
Right, it was a glitch in the matrix I think. Tried bulding it on a separate checkout of master, and it build `unittest_erasure_code_plugin_jerasure` just fine.
Clossing this PR.